### PR TITLE
change kidmissing error code

### PR DESCRIPTION
**Pull request by doodyparizada**

### CODE REVIEW

File: scripts/src/errors.ts [hunk 1]
```diff
@@ -14,34 +14,34 @@ const CODES = {
 		MissingToken: 1,
 		InvalidToken: 2,
 		InvalidApiKey: 3,
-		JwtKidMissing: 4,
-		TOSMissingOrOldToken: 5
+		TOSMissingOrOldToken: 5,
```
Comment (OrenZak) [inline scripts/src/errors.ts]: shouldn't be 4 ?

File: scripts/src/errors.ts [hunk 2]
```diff
@@ -14,34 +14,34 @@ const CODES = {
...
 	},
 	BadRequest: {
 		UnknownSignInType: 1,
 		WrongJwtAlgorithm: 2,
-		InvalidJwtSignature: 3,
-		InvalidPollAnswers: 4,
-		InvalidExternalOrderJwt: 5
+		InvalidPollAnswers: 3,
+		InvalidExternalOrderJwt: 4,
+		InvalidJwtSignature: 5,
```
Comment (OrenZak) [inline scripts/src/errors.ts]: `InvalidJwtSignature` is not written on the Wiki

File: scripts/src/errors.ts (hunk 1 above)
Comment (doodyparizada) [reply scripts/src/errors.ts]: I didn't want to break the current number. Are you or @Elaz  using this error?

File: scripts/src/errors.ts (hunk 2 above)
Comment (doodyparizada) [reply scripts/src/errors.ts]: added

File: scripts/src/errors.ts (hunk 1 above)
Comment (OrenZak) [reply scripts/src/errors.ts]: nop, you can change

**Pull request merged**